src/test/fixtures/sample-data-file-ebl.json

```diff
@@ -1,16 +1,20 @@
 {
+  "ownership": {
+    "beneficiaryAddress": "0xa61b056da0084a5f391ec137583073096880c2e3",
+    "holderAddress": "0xa61b056da0084a5f391ec137583073096880c2e3"
+  },
```

Comment:
Let's not put it in the same level as the data since there is a chance for collision of the `ownership` key in this case. It's a better practice to nest it one more layer to provide additional metadata for it, ie:

```
{
  data: {...dataGoesHere},
  ownershipe: {..ownershipDataGoesHere}
}
```

There will inevitably be changes to quite a bit of code everywhere but that will be easier for us to add new things in the data file the next time also.